### PR TITLE
Added Dungeon.java via upload

### PR DESCRIPTION
**Pull request by anguyenq**

### CODE REVIEW

File: Dungeon.java [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+		return sb.toString();
+	}
+	
+	public void moveHero() {
+		/*
+		 * PLACEHOLDER
+		 * - Will need to update row 4 of myLocales to keep track of how the
+		 * hero is moving
+		 * 
+		 * - idea: accept a number and use a switch to alter the coordinates
```
Comment (cwatss) [inline Dungeon.java:258]: I like the idea of using a switch here for the movement- will be much cleaner than if-else statements

Comment (anguyenq) [reply Dungeon.java:258]: I'm currently thinking of utilizing 2, 4, 6, 8 for the cardinal directions (based off of a normal numpad).

File: Dungeon.java [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+		 * 1 - Exit
+		 * 2 - Crown1
+		 * 3 - Crown2
+		 * 4 - Hero (Starts out at the entrance)
+		 */
+		myLocales = setLocales();
+		myMaze = setMaze();
+	}
+	
+	/**
```
Comment (cwatss) [inline Dungeon.java:58]: Just clarifying- this hashmap is keeping track of where the character has already been inside the maze, correct?

File: Dungeon.java (hunk 1 above)
Comment (anguyenq) [reply Dungeon.java:258]: Or WASD, whatever we end up deciding.

File: Dungeon.java (hunk 2 above)
Comment (anguyenq) [reply Dungeon.java:58]: No, it's keeping track of what coordinates have been used as it determines a potential location for each item of interest (entrance, exit, crown piece, etc.) as they are mutually exclusive.

**Pull request merged**